adapters/seedingAlliance/seedingAlliance.go

```diff
@@ -0,0 +1,142 @@
+package seedingAlliance
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type SeedingAllianceAdapter struct {
```

Comment:
Please rename to `type adapter struct {`, there is no need to duplicate a package name here.